setup.py

```diff
@@ -1,30 +1,30 @@
 #!/usr/bin/env python
 
 import os
-from setuptools import setup, find_packages
+
+from setuptools import find_packages, setup
 
 here = os.path.dirname(__file__)
 version_ns = {}
-with open(os.path.join(here, 'multiphonon', '_version.py')) as f:
+with open(os.path.join(here, "multiphonon", "_version.py")) as f:
```

Comment:
There is a story about conda package. I think it would be better to address this there.

Reply:
- For the sphinx stuff, I recommend we revisit this at the next tasks. One of them is about documentation (it is in the same story).
- The tests/data are excluded and reverted back just in case.
- End of file will be removed at the next push